src/Windows/Avalonia.Direct2D1/Media/ConicGradientBrushImpl.cs

```diff
@@ -0,0 +1,54 @@
+using System.Linq;
+using Avalonia.Media;
+
+namespace Avalonia.Direct2D1.Media
+{
+    public class ConicGradientBrushImpl : BrushImpl
```

Comment:
I don't think we should introduce duplicate code here. Instead reuse the existing RadialGradientImpl.

Reply:
I know what you mean and it felt messy - but the types IRadialGradientBrush and IConicGradientBrush are different so short of altering RadialGradientBrushImpl to reference IConicGradientBrush - which felt even messier - I couldn't see a particularly neat way of doing it. 

I did consider renaming ConicGradientBrushImpl to have Fake or something in it though?

Reply:
All types with `Impl` are considered private API. It is better to introduce a new ctor instead of duplication.

Reply:
I've removed the copy and paste implementation and made ConicGradients render using a new constructor on SolidColorBrushImpl using just the first stop for the colour.